src/views/ThePlatformPage.vue

```diff
@@ -15,7 +15,9 @@ import { createClock, incrementClock } from "@/domain/clock";
 import { createInitialPopulation } from "@/domain/people";
 
 const FLOOR_COUNT = 333;
-const MAX_PEOPLE_CAPACITY = FLOOR_COUNT * 2;
+const INITIAL_PEOPLE_PER_FLOOR = 2;
+const MAX_PEOPLE_CAPACITY = FLOOR_COUNT * INITIAL_PEOPLE_PER_FLOOR;
+const UNFOODABLE_FLOOR_COUNT = 51;
```

Comment:
Should we not base this var on how much previous people eat? As an example, if floors 0 to 50 aren't eating for 1 round?